sales.html

```diff
@@ -0,0 +1,23 @@
+<! DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+    <link href="css/reset.css" type="text/css" rel="stylesheet">
+    <link href="css/stylesales.css" type="text/css" rel="stylesheet">
+
+  </head>
+  <body>
+    <h1>Cookie Sales Locations</h1>
+    
+      <ul id="salesinpike">Cookies Sales in First and Pike</ul>
+      <ul id="seaTacAir">Cookies Sales in SeaTac Airport</ul>
+      <ul id="seattleCen">Cookies Sales in Seattle Center</ul>
+      <ul id="capHill">Cookies Sales in Seattle Center</ul>
```

Comment:
Make sure these last two have accurate titles, instead of all saying "Seattle Center"!